.github/workflows/ci_cron_weekly.yml

```diff
@@ -171,3 +170,24 @@ jobs:
             ASTROPY_USE_SYSTEM_ALL=1 pip3 install -v --no-build-isolation -e .[test]
```

Comment:
No, those were only needed because we were trying to also link against the system libraries I think to speed up compilation, but no reason we need to do this here. The remaining archs are still good tests of that external linking system.